SketchUp/Geom/Point3d.rb

```diff
@@ -416,13 +420,13 @@ def on_plane?(plane)
   #
   # @example
   #   line = [Geom::Point3d.new(0,0,0), Geom::Vector3d.new(0,0,1)]
-  #   point = Geom::Point3d.new 10,10,10
-  #   projected_point = point.project_to_line line
+  #   point = Geom::Vector3d.new(10,10,10)
```

Comment:
There are several more cases in this file where Point3d have been incorrectly renamed to Vector3d. I won't mark them all.